circle.yml

```diff
@@ -15,4 +15,5 @@ test:
       - swift build
   post:
       - bash <(curl -s https://codecov.io/bash)
+      - gem install jazzy && jazzy --clean
```

Comment:
Maybe we can add jazzy for the `Gemfile` as a source of all `Sourcery` ruby dependencies 😉